Formula/glib.rb

```diff
@@ -17,15 +17,19 @@ class Glib < Formula
     sha256 x86_64_linux:   "d896a5a0f3b720819fd86a85a95c7385b50212918ef9d7db8adb5750e2c2ed99"
   end
 
+  depends_on "gettext" => :build
```

Comment:
If available at build time, will be used to generate MO files. Removing `gettext` entirely results in not generating/installing `share/locale/*/LC_MESSAGES/glib20.mo`.

Runtime dependency is for systems that need libintl like macOS. GNU/Linux uses glibc instead.